regression/cstd/printf_5_bug/main.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <assert.h>
+
+struct D
+{
+  int dummy;
+  char fmt[16];
+};
+
+int main()
+{
+  unsigned n = nondet_uint() % 1024;
+  n++;
+  unsigned i = nondet_uint() % n;
+  struct D data[1];
+#ifndef _WIN32
+  int r = printf("%02X", data[0].fmt);
+  assert(r < 2);
+#else
+  assert(0);
```

Comment:
Do you know what happened here on Windows?

Reply:
Might be the same bug mentioned in #1155 ?
> ```sh
> no body for function __acrt_iob_func
> ```

pirntf_5_bug win log
```sh
Starting Bounded Model Checking
no body for function __builtin_va_start
no body for function __acrt_iob_func
Obtained empty format string.
no body for function __builtin_va_end
Symex completed in: 0.000s (32 assignments)
```

It seems `get_string_argument()` return an empty string